### PR TITLE
Publish year in review blog post

### PR DESCRIPTION
**Pull request by nikku**
Preview available [here](https://stage.bpmn.io/blog/posts/2021-year-in-review.html).

### CODE REVIEW

File: src/blog/2021/006-2021-year-in-review.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  <img src="{{ assets }}/attachments/blog/2018/014-christmas-tree.gif" alt="Merry Christmas to whom it may concern!" style="border-color: #14966396">
+  <p class="caption">
+    It is that time of the year again!
+  </p>
+</div>
+
+
+## The Year in Features
+
+Early this year [form-js](https://bpmn.io/toolkit/form-js/) joined the bpmn.io project family. With four follow up minor releases this represents our minimal take on for editing and embedding, powered by bpmn.io. With form-js we acknowledge the fact that, along with BPMN and DMN, forms are an [integral part of process communication](https://bpmn.io/blog/posts/2021-form-js-visual-form-editing-and-embedding.html).
```
Comment (MaxTru) [inline src/blog/2021/006-2021-year-in-review.md:33]: To make it easier to understand, consider

```suggestion
Early this year [form-js](https://bpmn.io/toolkit/form-js/) joined the bpmn.io project family. It also retrieved four follow up minor releases and with those, represents our minimal take on form editing and embedding, powered by bpmn.io. With form-js we acknowledge the fact that, along with BPMN and DMN, forms are an [integral part of process communication](https://bpmn.io/blog/posts/2021-form-js-visual-form-editing-and-embedding.html).
```

File: src/blog/2021/006-2021-year-in-review.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+* Released in January, [`bpmn-js@8`](https://github.com/bpmn-io/bpmn-js/blob/develop/CHANGELOG.md#800) significantly improves diagram mouse and keyboard navigation.
+* In March, [form-js](https://bpmn.io/toolkit/form-js/) joins as our take on visual form editing and embedding.
+* Released in May, [`bpmn-js@8.5`](https://github.com/bpmn-io/bpmn-js/blob/develop/CHANGELOG.md#850) significantly improves the message flow editing experience.
+* Released in June, [form-js@0.1](https://bpmn.io/blog/posts/2021-form-js-visual-form-editing-and-embedding.html) is our first stable form library release.
+* Also released in June, [`bpmn-js@8.7`](https://github.com/bpmn-io/bpmn-js/blob/develop/CHANGELOG.md#870) ships native support for the BPMN in color specification. Along with that we ship a reworked [color picker extension](https://github.com/bpmn-io/bpmn-js-color-picker).
+* Also released in June, `dmn-js@v11.0` ships with [modern import and export APIs](http://localhost:9012/blog/posts/2021-dmn-js-11-0-awaitable-import-export-apis.html).
```
Comment (MaxTru) [inline src/blog/2021/006-2021-year-in-review.md:42]: Why no link to dmn-js?

File: src/blog/2021/006-2021-year-in-review.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+Significant work happened in one of the many underlying libraries that support our editing ecosystem. As an example, we [published 20 releases](https://www.npmjs.com/package/diagram-js?activeTab=versions) of [diagram-js](https://github.com/bpmn-io/diagram-js/blob/develop/CHANGELOG.md), each improving the way our core modeling UX works.
+
+We cut more than 95 releases (more than 40 being actual feature releases) in [over 30 libraries](https://github.com/bpmn-io). To communicate the new and noteworthy in our ecosystem, we published [five blog posts](https://bpmn.io/blog/) and sent out [numerous tweets](https://twitter.com/bpmn_io).
+
+
+## The Year in Numbers
+
+Similar to [previous](./2018-year-in-review.html) [years](./2019-year-in-review.html) we see significant increase in interest in our project and toolkits.
```
Comment (MaxTru) [inline src/blog/2021/006-2021-year-in-review.md:59]:
```suggestion
Similar to [previous](./2018-year-in-review.html) [years](./2019-year-in-review.html) we were able to significantly increase the interest in our projects and toolkits.
```

File: src/blog/2021/006-2021-year-in-review.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+More people visited [our project website](https://bpmn.io/), asked for help, and contributed to [our forum](https://forum.bpmn.io/) and starred our projects on [GitHub](https://github.com/bpmn-io). Community work led to several contributions, too, such as [spanish translations for bpmn-js](https://github.com/bpmn-io/bpmn-js-i18n/pull/11) and [improved](https://github.com/bpmn-io/diagram-js/pull/484) [keyboard accessibility](https://github.com/bpmn-io/diagram-js/pull/493) of our editors.
```
Comment (MaxTru) [inline src/blog/2021/006-2021-year-in-review.md:60]:
```suggestion
More people visited [our project website](https://bpmn.io/), asked for help, contributed to [our forum](https://forum.bpmn.io/), and starred our projects on [GitHub](https://github.com/bpmn-io). Community work led to several contributions, too, such as [spanish translations for bpmn-js](https://github.com/bpmn-io/bpmn-js-i18n/pull/11) and [improved](https://github.com/bpmn-io/diagram-js/pull/484) [keyboard accessibility](https://github.com/bpmn-io/diagram-js/pull/493) of our editors.
```

File: src/blog/2021/006-2021-year-in-review.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+<div class="figure no-border condensed-size" style="margin: 50px 0 40px 0; text-align: center">
+  <img src="{{ assets }}/attachments/blog/2021/006-stats.png" alt="Some project statistics" style="max-width: 720px">
```
Comment (MaxTru) [inline src/blog/2021/006-2021-year-in-review.md:63]: Consider to make the increased bundle size grey in colour

File: src/blog/2021/006-2021-year-in-review.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+## The Year as a Collaborative Effort
+
+With two more folks on board in 2021, we are now at a respectable team of seven developers, dedicated to working on our toolkits as well as the core Camunda modeling apps. Head over to [our about page](https://bpmn.io/about/) to learn more about us.
+
+
+## Looking Ahead <a name="looking-ahead"></a>
+
+Making great BPM tooling available everywhere for everyone is a mission, not a task on a checklist. Considering this, our future could not be more exciting. So what is up next?
+
+In 2022 we are going to ship a full rework of our properties panel infrastructure, and new properties panels for BPMN, DMN, and forms. Async data loading will be easily possible. Promised <img class="emoji" src="https://twemoji.maxcdn.com/v/13.1.0/svg/1f91e-1f3fc.svg" alt="crossed fingers">.
```
Comment (MaxTru) [inline src/blog/2021/006-2021-year-in-review.md:81]: Consider link to bpmn-js-properties-panel

**Review by MaxTru — COMMENTED** [review]
Very nice

File: src/blog/2021/006-2021-year-in-review.md (hunk 2 above)
Comment (nikku) [reply src/blog/2021/006-2021-year-in-review.md:42]: :+1:

File: src/blog/2021/006-2021-year-in-review.md (hunk 5 above)
Comment (nikku) [reply src/blog/2021/006-2021-year-in-review.md:63]: Why?

**Comment by nikku** [comment]
Updated. If you want to, have another look.

File: src/blog/2021/006-2021-year-in-review.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+* Released in January, [`bpmn-js@8`](https://github.com/bpmn-io/bpmn-js/blob/develop/CHANGELOG.md#800) significantly improves mouse and keyboard navigation.
+* In March, [form-js](https://bpmn.io/toolkit/form-js/) joins as our take on visual form editing and embedding.
+* Released in May, [`bpmn-js@8.5`](https://github.com/bpmn-io/bpmn-js/blob/develop/CHANGELOG.md#850) significantly improves the message flow editing experience.
+* Released in June, [form-js@0.1](./2021-form-js-visual-form-editing-and-embedding.html) is our first stable form library release.
+* Also released in June, [`bpmn-js@8.7`](https://github.com/bpmn-io/bpmn-js/blob/develop/CHANGELOG.md#870) ships native support for the [BPMN in color](https://github.com/bpmn-miwg/bpmn-in-color) specification. Along with that, we published a reworked [color picker extension](https://github.com/bpmn-io/bpmn-js-color-picker).
+* Also released in June, [`dmn-js@v11.0`](https://github.com/bpmn-io/dmn-js/blob/develop/packages/dmn-js/CHANGELOG.md#1100) ships with [modern import and export APIs](./2021-dmn-js-11-0-awaitable-import-export-apis.html).
```
Comment (barmac) [inline src/blog/2021/006-2021-year-in-review.md:42]:
```suggestion
* Also released in June, [`dmn-js@11`](https://github.com/bpmn-io/dmn-js/blob/develop/packages/dmn-js/CHANGELOG.md#1100) ships with [modern import and export APIs](./2021-dmn-js-11-0-awaitable-import-export-apis.html).
```

File: src/blog/2021/006-2021-year-in-review.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+Early this year, [form-js](https://bpmn.io/toolkit/form-js/) joined the bpmn.io project family. It also retrieved four minor follow-up releases and, with those, represents our minimal take on form editing and embedding, powered by bpmn.io. With form-js, we acknowledge that, along with BPMN and DMN, forms are an [integral part of process communication](./2021-form-js-visual-form-editing-and-embedding.html).
+
+Other greater topics were modernizations of our code-base, improvement of standards compatibility, and modeling UX improvements. Some noteworthy core library milestones accomplished this year are the following:
+
+* Released in January, [`bpmn-js@8`](https://github.com/bpmn-io/bpmn-js/blob/develop/CHANGELOG.md#800) significantly improves mouse and keyboard navigation.
+* In March, [form-js](https://bpmn.io/toolkit/form-js/) joins as our take on visual form editing and embedding.
+* Released in May, [`bpmn-js@8.5`](https://github.com/bpmn-io/bpmn-js/blob/develop/CHANGELOG.md#850) significantly improves the message flow editing experience.
+* Released in June, [form-js@0.1](./2021-form-js-visual-form-editing-and-embedding.html) is our first stable form library release.
```
Comment (barmac) [inline src/blog/2021/006-2021-year-in-review.md:40]:
```suggestion
* Released in June, [`form-js@0.1`](./2021-form-js-visual-form-editing-and-embedding.html) is our first stable form library release.
```

File: src/templates/partials/hiring_footer.hbs [hunk 9]
```diff
@@ -1,7 +1,7 @@
 {{#markdown}}
 
 <p style="border-top: dotted 1px #CCC; margin-top: 40px; padding-top: 20px">
-Are you passionate about JavaScript, modeling and the web?
+Are you passionate about JavaScript, modeling, and the web?
```
Comment (barmac) [inline src/templates/partials/hiring_footer.hbs:4]: I don't quite get the call to action:

> Join Camunda and build modeling tools people <3

What do you build when you build modeling tools people? 🤔

**Comment by barmac** [comment]
Great blog post. I left a couple of small suggestions.

**Pull request merged**